Resources/Prototypes/Catalog/Cargo/cargo_vending.yml

```
@@ -22,7 +22,7 @@
     sprite: Objects/Specific/Service/vending_machine_restock.rsi
     state: base
   product: CrateVendingMachineRestockChefvendFilled
-  cost: 680
+  cost: 750 // Starlight-edit
```

```suggestion
  cost: 750 # Starlight-edit
```
Oh, sorry, this must to be as yaml comment

---

Oops, fixed in [`39f3ae8` (#1504)](https://github.com/ss14Starlight/space-station-14/pull/1504/commits/39f3ae8ffd35f50cb39d50a8b8c2d245d396aa5d)